apps/site/redirects.json

```diff
@@ -339,6 +339,10 @@
     {
       "source": "/:locale/discord",
       "destination": "https://discord.gg/vUsrbjd"
+    },
+    {
+      "source": "/esp/herodevs",
+      "destination": "https://www.herodevs.com/support/node-nes?utm_source=NodeJS+&utm_medium=Link&utm_campaign=Nodejs_eol_support"
```

Comment:
Is this working? I've tested it and it wasn't. BTW, +1 on tracking it in this way.